client/packages/common/src/intl/locales/en/common.json

```diff
@@ -141,6 +139,7 @@
   "label.items-selected_other": "{{count}} items selected",
   "label.line": "Line {{line, number}}",
   "label.line-total": "Line total",
+  "label.label": "Label",
```

Comment:
tbh - I don't find `Label` helpful - it doesn't tell me what the intention of the column is! Is `Status` not correct here?

Reply:
The encounter list already has a status column, I could somehow merge them but in general there could be multiple statuses valid at the same time...